.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,40 @@
+name: Bug report
+description: Create a report to help us improve QMK Firmware.
+title: "[Bug] "
+labels: ["bug", "help wanted"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Provide a general summary of the bug in the title above.
+  - type: textarea
+    attributes:
+      label: Describe the Bug
+      description: A clear and concise description of what the bug is.
+  - type: input
+    attributes:
+      label: Keyboard Used
+  - type: input
+    attributes:
+      label: Revision (if applicable)
```

```suggestion
      label: Link to product page (if applicable)
```
I think this might be a better data point, tbh.